target/linux/x86/image/64.mk

```
@@ -0,0 +1,5 @@
+define Device/generic
```

```suggestion
define Device/generic-x86_64
```

This way the x86 target would have unique IDs for PROFILES. I'd be in favor of this. So you can just specify the desired PROFILE to build and the target is deterministic